<?xml version="1.0"?>
<launch>
  <arg name="razor_config_file_1" default="$(env HOME)/catkin_ws/src/race/launch/IMU/my_razor.yaml"/>
  <node pkg="razor_imu_9dof" type="imu_node.py" name="imu_node_1" output="screen">
    <rosparam file="$(arg razor_config_file_1)" command="load"/>
    <remap from="imu" to="imu_1/data_raw"/>
  </node>
  <node pkg="imu_filter_madgwick" type="imu_filter_node" name="imu_filter_node_1" 
    output="screen">
    <param name="use_mag" value="false"/>
    <remap from="imu/data_raw" to="imu_1/data_raw"/>
    <remap from="imu/data" to="imu_1"/>
  </node>
  <node pkg="race" type="imu_tf.py" name="imu_tf_1" output="screen">
  	<remap from="imu_topic" to="imu_1"/>
    <remap from="imu_transformed" to="imu_transformed_1"/>
  </node>
</launch>
